sdk/rust/oak/src/logger/mod.rs

```diff
@@ -53,17 +35,15 @@ impl Log for OakChannelLogger {
         if !self.enabled(record.metadata()) {
             return;
         }
-        let log_entry = LogEntry {
-            // We add a newline to the message to force flushing when printed by the host.
-            message: format!(
-                "{}  {} : {} : {}\n",
-                record.level(),
-                record.file().unwrap_or_default(),
-                record.line().unwrap_or_default(),
-                record.args()
-            ),
+        let log_msg = crate::proto::logger::LogMessage {
+            file: record.file().unwrap_or_default().to_string(),
```

Comment:
Perhaps make this more explicit, e.g. `unwrap_or("<unknown>")` or similar.

Reply:
Changed it to `file: record.file().unwrap_or("<unknown-file>").to_string(),`. 
The next line also calls `unwrap_or_default()`, but on an int value. Do we need an explicit default value there too?

Reply:
I guess that's enough, obviously it will show up as `0` in the output, but I guess that's fine for now.